src/providers/ASAModuleUpdateCodeLensProvider.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+"use strict";
+
+import * as jsonMap from "json-source-map";
```

Comment:
we already have "jsonc-parser" dependency. the lib cannot work?

Reply:
jsonc-parser is used for token parse, it's location is the total character number, not the line number and column number, which is is not suitable in our scenario